src/main/java/com/google/sps/data/Article.java

```diff
@@ -20,14 +20,14 @@
 /** A well-being related article that will be displayed */
 public final class Article {
 
-  private final ArrayList<String> source;
+  private final String source;
   private final String author;
   private final String title;
   private final String description;
   private final URL url;
   private final String publishedAt;
```

Comment:
What is this field variable for? The name of the publisher or the time it was published? Use a more descriptive name. Also if it's suppose to be the date/time it was published, It would be better to use Timestamp instead of a String.

Reply:
Yes, publisher would be a better name.

Reply:
The date/time is given by the API as a String so we would have to add some kind of function to convert it to a timestamp in long, do you guys want to do that?

Reply:
maybe for now, we can leave it as it is, and come back to it during week 9